src/main/kotlin/com/sourcegraph/cody/initialization/PostStartupActivity.kt

```diff
@@ -29,7 +28,6 @@ class PostStartupActivity : StartupActivity.DumbAware {
     if (ConfigUtil.isCodyEnabled()) CodyAgentService.getInstance(project).startAgent(project)
     CodyAutocompleteStatusService.resetApplication(project)
     CodyFocusChangeListener().runActivity(project)
-    CodyAgentCodebase.getInstance(project).onFileOpened(project, null)
```

Comment:
iiuc this call has been moved to CodyAgentCodebase

Reply:
Correct.